server/src/internalClusterTest/java/org/opensearch/nodestats/NodeStatsIT.java

```diff
@@ -243,6 +250,53 @@ public void testNodeIndicesStatsDocStatusStatsCreateDeleteUpdate() {
         }
     }
 
+    public void testNodeIndicesStatsResponseOptimised() {
```

Comment:
Can you also add some mixed cluster tests to test the compatibility between different versions?